ocaml/parsing/lexer.mll

```diff
@@ -80,6 +80,7 @@ let keyword_table =
     "once_", ONCE;
     "open", OPEN;
     "or", OR;
+    "overwrite_", OVERWRITE;
```

Comment:
Before merging to `main`, we'll need to check that there are no variables named `overwrite_` in `jane`.

Reply:
Is there a CR for this little piece?

Reply:
There is now :)